.travis.yml

```diff
@@ -2,47 +2,85 @@ sudo: false
 language: ruby
 script: bin/test
 install: bin/bootstrap --path vendor/bundle --without development debug
-rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.10
-  - 2.2.7
-  - 2.3.4
-  - 2.4.1
-  - ruby-head
-  - jruby
-  - jruby-20mode
-  - jruby-21mode
-  - jruby-9.1.12.0
 matrix:
   include:
+    - rvm: ruby-head
+      os: linux
+    - rvm: ruby-head
+      os: osx
```

Comment:
Seeing the result of Travis, ruby-head (osx) environment in Travis server has problem right now.
You may be able to remove the ruby-head (osx) if you do not like to get the error every time.
https://travis-ci.org/cucumber/aruba/jobs/259520443